integration-tests/framework.yaml

```diff
@@ -8,8 +8,8 @@ logging:
   level: 0
 
 # Specify the image and version of the chainlink image you want to run tests against. Leave blank for default.
-chainlink_image:
-chainlink_version:
+chainlink_image: public.ecr.aws/chainlink/chainlink
+chainlink_version: 1.4.1
```

Comment:
What is happening here? Is this targetting the last released version, as opposed to current develop or branch state?

Reply:
This is setting the default version to test as the latest good release. This is overridden in CI to use the latest built image for the PR.